### PR TITLE
chore: add dev mail list and rename ceresdb to horaedb

### PR DESCRIPTION
**Pull request by tanruixiang**
## Rationale
Add dev mail list for dicussing
Related https://github.com/apache/incubator-horaedb/issues/1319 `Update all CeresDB URLs after repo transferred`
## Detailed Changes
Add dev mail in readme and rename org name

## Test Plan
N/A

### CODE REVIEW

**Comment by tanruixiang** [comment]
@jiacai2050 Please help to check whether it can be subscribe successfully.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ HoraeDB is a high-performance, distributed, cloud native time-series database.
 ## Contributing
 Any contribution is welcome!
 
+Discuss at [dev mailing list](mailto:dev-subscribe@horaedb.apache.org) ([subscribe](mailto:dev-subscribe@horaedb.apache.org?subject=(send%20this%20email%20to%20subscribe)) / [unsubscribe](mailto:dev-unsubscribe@horaedb.apache.org?subject=(send%20this%20email%20to%20unsubscribe)) / [archives](https://lists.apache.org/list.html?dev@horaedb.apache.org))
```
Comment (jiacai2050) [inline README.md:19]:
```suggestion
Discuss at [dev mailing list](mailto:dev-subscribe@horaedb.apache.org) ([subscribe](mailto:dev-subscribe@horaedb.apache.org) / [unsubscribe](mailto:dev-unsubscribe@horaedb.apache.org) / [archives](https://lists.apache.org/list.html?dev@horaedb.apache.org))
```

**Comment by caicancai** [comment]
You may also need to update REAME-CN

File: README.md (hunk 1 above)
Comment (tanruixiang) [reply README.md:19]: Maybe some clients don't support empty content, so non-empty content is used here.

**Comment by tanruixiang** [comment]
> You may also need to update REAME-CN #1342

Thank you, done! Welcome to subscribe to our mailing list to ask question or start discussion.

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Comment by tanruixiang** [comment]
> LGTM

@waynexia @tisonkun Please help merge if you have time.

**Review by tisonkun — APPROVED** [review]

**Comment by waynexia** [comment]
I'm late, feel free to ping me if there is any in the future

**Comment by tanruixiang** [comment]
> I'm late, feel free to ping me if there is any in the future

Thank you! ❤️

**Pull request merged**